addons/io_scene_gltf2/blender/com/gltf2_blender_math.py

```diff
@@ -159,3 +167,8 @@ def transform_scale(scale: Vector, transform: Matrix = Matrix.Identity(4)) -> Ve
 def transform_value(value: Vector, _: Matrix = Matrix.Identity(4)) -> Vector:
     """Transform value."""
     return value
+
+
+def round_near(value: float, target: float) -> float:
```

Comment:
I think `round_if_near` would be a better name

Reply:
That makes sense — done.